backends/avx/ceed-avx-tensor.c

```diff
@@ -35,43 +52,43 @@ static inline int CeedTensorContract_Avx_Blocked(CeedTensorContract contract, Ce
     // Blocks of 4 rows
     for (CeedInt j = 0; j < (J / JJ) * JJ; j += JJ) {
       for (CeedInt c = 0; c < (C / CC) * CC; c += CC) {
-        __m256d vv[JJ][CC / 4];  // Output tile to be held in registers
+        rtype vv[JJ][CC / 4];  // Output tile to be held in registers
```

Comment:
Should this magic 4 also be updated to be 4 doubles or 8 floats? (I should have made this change ages ago but forgot about it)